src/Ormolu/Printer/Meat/Declaration/Warning.hs

```diff
@@ -34,10 +34,11 @@ p_topLevelWarning :: [LocatedN RdrName] -> WarningTxt -> R ()
 p_topLevelWarning fnames wtxt = do
   let (pragmaText, lits) = warningText wtxt
   switchLayout (fmap getLocA fnames ++ fmap getLoc lits) $
-    pragma pragmaText . inci $ do
-      sep commaDel p_rdrName fnames
-      breakpoint
-      p_lits lits
+    pragma pragmaText . inci $
+      do
```

Comment:
We need to check whether this is the expected behaviour or not (`$` chains with 2+ `$` and a hanging last operand)
We should also add test cases for this.

Reply:
`do` should stay in the hanging position.

Reply:
Fixed with my last commit